serivce-chain-azure/Exist-Vnet/en.tf

```diff
@@ -0,0 +1,152 @@
+# refer to a resource group
+data "azurerm_resource_group" "en" {
+  name = "${var.resource_group_name}"
```

Comment:
Since Terraform 0.12 you can set variables directly like this:

```suggestion
  name = var.resource_group_name
```

Terraform can sometimes update this if you run `terraform 0.12upgrade`, but it doesn't always work if 0.12 features are already used. (I tested, and it seems to work for the top-level module, but not all the submodules.)

The `"${var}"` form was deprecated in 0.12 so Terraform will generate a warning when running `terraform apply`.

Reply:
Hello @markyim-gx,

Thanks for updating this field. There are still a lot of other places which use this format, do you plan to update those as well, or will you address them in a separate PR?

Reply:
updated ! :)